misp/config.yml.sample

```diff
@@ -0,0 +1,15 @@
+rabbitmq:
+  hostname: 'localhost'
+  port: 5672
+  username: 'guest'
+  password: 'guest'
+
+misp:
```

Comment:
Could be "connector:" to standardize the configuration no?

Reply:
I prefer keep this way to have a more comprehensive link to environment variables:

RABBITMQ_HOSTNAME => config['rabbitmq']['hostname']
MISP_URL => config['misp']['url']

And I think it is more understandable to have MISP_URL and MISP_KEY in the docker-compose.yml than CONNECTOR_URL and CONNECTOR_KEY.